specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/virtualWan.json

```diff
@@ -2593,7 +2901,330 @@
+        },
+        "vpnAuthenticationTypes": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "description": "VPN authentication types enabled for the VpnServerConfiguration.",
+            "enum": [
+              "Certificate",
+              "Radius",
+              "AAD"			  
+            ],
+            "x-ms-enum": {
+              "name": "VpnAuthenticationType",
+              "modelAsString": true
+            }
+          },
+          "description": "VPN authentication types for the VpnServerConfiguration."
+        },		
+        "vpnServerConfigVpnClientRootCertificates": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/VpnServerConfigVpnClientRootCertificate"
+          },
+          "description": "VPN client root certificate of VpnServerConfiguration."
+        },
+        "vpnServerConfigVpnClientRevokedCertificates": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/VpnServerConfigVpnClientRevokedCertificate"
+          },
+          "description": "VPN client revoked certificate of VpnServerConfiguration."
+        },
+        "vpnServerConfigRadiusServerRootCertificates": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/VpnServerConfigRadiusServerRootCertificate"
+          },
+          "description": "Radius Server root certificate of VpnServerConfiguration."
+        },
+        "vpnServerConfigRadiusClientRootCertificates": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/VpnServerConfigRadiusClientRootCertificate"
+          },
+          "description": "Radius client root certificate of VpnServerConfiguration."
+        },
+        "vpnClientIpsecPolicies": {
+          "type": "array",
+          "items": {
+            "$ref": "./virtualNetworkGateway.json#/definitions/IpsecPolicy"
+          },
+          "description": "VpnClientIpsecPolicies for VpnServerConfiguration."
+        },
+        "radiusServerAddress": {
+          "type": "string",
+          "description": "The radius server address property of the VpnServerConfiguration resource for point to site client connection."
+        },
+        "radiusServerSecret": {
+          "type": "string",
+          "description": "The radius secret property of the VpnServerConfiguration resource for point to site client connection."
+        },
+        "aadAuthenticationParameters": {
+          "$ref": "#/definitions/AadAuthenticationParameters",
+          "description": "The set of aad vpn authentication parameters."
+        },
+        "provisioningState": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The provisioning state of the VpnServerConfiguration resource. Possible values are: 'Updating', 'Deleting', and 'Failed'."
+        },
+        "p2SVpnGateways": {
```

Comment:
Friendly reminder to make sure linked access checks are defined in your manifest to ensure proper authorization.

Reply:
Sure, I will update maifest PR with linked access checks.